docs/Time_in_Arbitrum.md

```diff
@@ -0,0 +1,48 @@
+# Time in Arbitrum Rollup
+
+As on Ethereum, Arbitrum doesn't use real wall-clock time, but instead uses block numbers, which correspond to block numbers on the main Ethereum chain.
+
+Also as on Ethereum, clients submit transactions they want to see happen, and the system (usually) executes those transactions at some later time.
+In Arbitrum Rollup, clients submit transactions by posting messages to the Ethereum chain.
+These messages are put into the chain's *pending inbox*.
+Every message is timestamped with the time it was put into the pending inbox.
+
+Messages in an ArbChain's pending inbox are processed in order.
+Generally, some time will elapse between the time when a message is put into the pending inbox (and timestamped) and the time when the contract processes the message and carries out the transaction requested by the message.
+
+If your Solidity contract, running on Arbitrum Rollup, accesses ``block.number``, this will give you the ArbChain's current time. 
+If you want to know when the current message (the one that requested the transaction you're currently running) was timestamped, you can get it in Solidity by writing:
+
+    ArbSys(address(100)).currentMessageTime()
```

Comment:
Somewhere we should separately explain ArbSys calls since the syntax is weird

Reply:
Maybe another documentation page about ArbSys?

Reply:
I'm creating an issue to document ArbSys.